addOns/pscanrules/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Maintenance changes.
 - The alerts of the Hash Disclosure scan rule no longer have the evidence duplicated in the Other Info field.
 
+### Fixed
+- The  Absence of Anti-CSRF Tokens Scan Rule now skips responses that are not HTML (Issue 7890)
```

Comment:
Looks like there's a double space after the "The", sorry I missed that earlier.

Reply:
My bad. Do you have a plugin recommendation for Intellij IDEA to prevent these mistakes?